input/examples/practitioner-chau-fryer.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Practitioner xmlns="http://hl7.org/fhir">
+    <id value="chau-fryer"/>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
+    </meta>
+    <name>
+        <family value="FRYER" />
+        <given value="Chau" />
+    </name>
+    <telecom>
+        <system value="phone"/>
+        <value value="(03) 7010 8625"/>
```

Comment:
Replace with the following mobile number 0491572665

Reply:
Done